src/main/java/tetris/Tetromino.java

```diff
@@ -198,4 +198,36 @@ public Tetromino rotateClockwise(Tetromino shape) {
     }
 
     //add rotateCounter()
+
+    /**
+     * @return last element/bottom of shape.
+     * Used if the shape reached bottom of grid or other shape at bottom.
+     */
+    public Square getShapeBottom(Tetromino shape) {
+        if (oShape.equals(shape)) {
```

Comment:
TODO: make equals method

Reply:
Also we need a currently falling shape somehow.

Reply:
added an equals method to Square